Duke.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class Duke {
+
+    private static int Count = 0;
+    private static Task[] List = new Task[10]; //creating list of Task for inputs
```

Comment:
Hi, as mentioned by the prof during the lecture, 10 is a magic number, and reader like me may not know why the number is set to 10 instead of 100 or else. So you may give the number a meaning by defining it as a constant. Hope this helps!